src/Analyzer/Resolve/IdentifierResolver.cpp

```diff
@@ -733,7 +947,39 @@ IdentifierResolveResult IdentifierResolver::tryResolveIdentifierFromTableExpress
     }
 
     if (identifier.getPartsSize() == 1)
```

Comment:
Cases in the bottom use table and database names and they are not handled.

Reply:
Following function in the top would break:
* `table_expression_data.hasFullIdentifierName(IdentifierView(identifier))`
* `table_expression_data.canBindIdentifier(IdentifierView(identifier))`